addons/mail/static/src/components/composer/composer.xml

```diff
@@ -97,18 +97,19 @@
                                         'o-open': composerView.emojisPopoverView,
                                         'o-mobile': messaging.device.isMobile,
                                     }"
+                                    t-att-aria-expanded="composerView.emojisPopoverView ? 'true' : 'false'"
                                     t-on-keydown="composerView.onKeydownButtonEmojis"
                                     t-on-click="composerView.onClickButtonEmojis"
                                     t-ref="buttonEmojis"
                                 >
-                                    <i class="fa fa-smile-o"/>
+                                    <i class="fa fa-smile-o" role="img" aria-label="Emojis"/>
```

Comment:
Also note this was only added in this version (aria-label was on the button on previous version)?

Reply:
It has the same effect adding it to the button as adding it to the image, but the latter is semantically more correct as the image is the one giving information and the button has no title.

Reply:
It's fine I was just pointing the differences I noticed to be sure.